src/parser/wsdl.js

```diff
@@ -203,10 +225,22 @@ class WSDL {
           deepMerge(self.definitions.schemas[include.namespace ||
           wsdl.definitions.$targetNamespace], wsdl.definitions);
       }
-      self._processNextInclude(includes, function(err) {
+      self._processNextInclude(includes, function (err) {
         callback(err);
       });
-    });
+    };
+
+    if (self.syncLoad) {
+      WSDL.loadSync(includePath, options, function (err, wsdl) {
```

Comment:
I think `loadSync` should not take in the callback function since it's not async. Maybe I'm missing something. Also same comment for `openSync`.

Reply:
Agreed I'll update too with @jonrober-80 :-)